docs/7.x/pack.md

```diff
@@ -816,7 +816,7 @@ local Docker registry so that when Helm renders resource templates, they contain
 correct image references.
 
 !!! tip
-    There is a sample application available on [GitHub](https://github.com/gravitational/quickstart/tree/master/mattermost)
+    There is a sample application available on [GitHub](https://github.com/gravitational/gravity/tree/master/examples/wordpresshttps://github.com/gravitational/gravity/tree/master/examples/wordpress)
```

Comment:
There's a redundant suffix in the URL: https://github.com/gravitational/gravity/tree/master/examples/wordpress, it should be just:
```
There is a sample application available on [GitHub](https://github.com/gravitational/gravity/tree/master/examples/wordpress).
```

Also, I found that linking the master branch has a tendency to rot (when things get moved around), so to avoid it it's better to reference a specific tag or commit SHA:
```
There is a sample application available on [GitHub](https://github.com/gravitational/gravity/tree/6048f1e994a14fb9b1b7fad16fdf56db96eef212/examples/wordpress)
```